app/controllers/catalog_controller.rb

```diff
@@ -66,13 +66,13 @@ class CatalogController < ApplicationController
     ## Default parameters to send on single-document requests to Solr. These settings are the Blackligt defaults (see SearchHelper#solr_doc_params) or
     ## parameters included in the Blacklight-jetty document requestHandler.
     #
-    #config.default_document_solr_params = {
-    #  qt: 'document',
-    #  ## These are hard-coded in the blacklight 'document' requestHandler
-    #  # fl: '*',
-    #  # rows: 1,
-    #  # q: '{!term f=id v=$id}'
-    #}
+    config.default_document_solr_params = {
```

Comment:
In upgrading tul_cob to solr8 I ran into a weird issue with the document handler being overridden with some default handler.  I wonder whether that's the same issue you ran into here.  This works though.  So I think we can keep and maybe change in the future.